lib/plugins/aws/deploy/lib/initializeResources.js

```diff
@@ -49,6 +49,18 @@ module.exports = {
     // check if the user has added some "custom Resources" (and merge them into coreCFTemplate)
     if (this.serverless.service.resources && this.serverless.service.resources.Resources) {
       forEach(this.serverless.service.resources.Resources, (value, key) => {
+        // check if the user has added some "custom Outputs"
+        if ('Outputs' in value) {
+          forEach(value.Outputs, (outputValue, outputKey) => {
+            const newOutputsObject = {
+              [outputKey]: outputValue,
+            };
+
+            merge(coreCFTemplate.Outputs, newOutputsObject);
+          });
+          delete value.Outputs; // eslint-disable-line no-param-reassign
```

Comment:
Not sure if this is preferable? But we really don't want those `Outputs` as part of the `Resource` 😊